codes/qubits/color code

```diff
@@ -0,0 +1,70 @@
+
+protection:
+'Color code is defined on specific kind of lattices to protect against errors. The code distance is related to the homology of string operators in the code.\cite{arXiv:1311.0277}'
+# This field starts a list of specific labeled subfields; do not leave it
+# empty. If empty, comment out. Also, indentations are important!
+features:
+  encoders:
+    - 'Stabilizer measurement \cite{arxiv:1404.2495}.'
+  transversal_gates: #'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+    - 'Universal transversal gate can be achieved in 3d color code using gauge fixing\cite{arXiv:1304.3709}\cite{arXiv:1403.2734}.'
+    - 'Color-code lattice surgery\cite{arXiv:1407.5103}.'
+    - 'Code deformation\cite{doi:10.1088/1367-2630/13/4/043005}\cite{arXiv:0806.4827}.'
+  general_gates:
+    - 'Magic-state distillation protocols \cite{doi:10.7907/059V-MG69}.'
+    - 'Syndrome measurement\cite{arXiv:1407.5103}.'
+    - 'CNOT can be implemented via braiding \cite{arXiv:0806.4827}.'
+    - 'Non-clifford gates can be implemented via code switching\cite{doi:10.7907/059V-MG69}.'
+  decoders:
+    - 'Sweep decoder shown to have a non-zero threshold \cite{doi:10.7907/059V-MG69}.'
+    - 'Projection decoder\cite{doi:10.7907/059V-MG69}.'
+    - 'Clustering decoder gives error rate of ~0.31% \cite{arXiv:1503.08217}.'
+    - 'Matching decoder gives low logical failure rate \cite{arXiv:2108.11395}.'
+    - 'Integer-program-based decoder\cite{arXiv:1402.3037}'
+    - 'Restriction decoder\cite{doi:10.1088/1367-2630/ab68fd}'
+  threshold:
+    - 'For 2d, \(\geq 6.25\)% threshold with error free syndrome extraction and 0.1% threshold error rate with faulty syndrome extraction \cite{	arXiv:0907.1708}.'
+    - 'For 3d, 0.46% with clustering decoder \cite{arXiv:1708.07131}.'
+    - 'For 3d, 1.9% for 1D string-like logical operators and 27.6% for 2D sheet-like operators with noise models using optimal decoding and perfect measurements\cite{arXiv:1708.07131}.'
+    - '0.31% noise threshold error rate for gauge code \cite{	arXiv:1503.08217}'
+    - '0.143% with depolarising circuit-level noise using perfect-matching decoder \cite{	arXiv:1407.5103}'
+
+#realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+#notes:
+  #- 'Color code is equivalent to toric code in many ways\cite{10.1088/1367-2630/17/8/083026}.'
+
+relations:
+  parents:
+    - code_id: css
+      detail: 'Color code is a self dual CSS code'
```

Comment:
Mention explicitly that stabs are products of only X or only Z Paulis. Omit "self-dual".